### ISSUE TITLE
Total Cost of Ownership

### ISSUE THREAD

**Issue by jqnatividad** [centerforgov/open-data-portal-requirements#11]
TCO should be a consideration, especially for smaller jurisdictions, who do not have the capacity/budget for expensive solutions.

If the data portal becomes fundamental infrastructure, and the govt starts "dog-fooding" it into their operational workflow and the local civic-hacker community builds solutions on top of it, this functionally locks-in the jurisdiction to the solution.

Apart from the cost of running the service, costs calculations should include:
- ability to configure/customize the data portal without having to sole-source professional services from one vendor
- cost of "optional" features which turn out to become "essential" when the data portal is fully utilized
- switching cost

**Comment by technickle** [comment]
I agree in principle, but I think this is outside the scope of a functional requirements document, and perhaps more suited to be a part of the guidance in the [Selecting the Right Technology](https://centerforgov.gitbooks.io/open-data-getting-started/content/selecting-the-right-technology.html) section of our [Open Data: Getting Started](https://www.gitbook.com/book/centerforgov/open-data-getting-started/details) guide.
